package.json

```diff
@@ -106,33 +106,33 @@
-    "@opentelemetry/instrumentation-dns": "~0.23.0",
-    "@opentelemetry/instrumentation-express": "~0.23.0",
-    "@opentelemetry/instrumentation-graphql": "~0.23.0",
-    "@opentelemetry/instrumentation-grpc": "~0.23.0",
-    "@opentelemetry/instrumentation-hapi": "~0.23.0",
-    "@opentelemetry/instrumentation-http": "~0.23.0",
-    "@opentelemetry/instrumentation-ioredis": "~0.23.0",
-    "@opentelemetry/instrumentation-knex": "~0.23.0",
-    "@opentelemetry/instrumentation-koa": "~0.23.0",
-    "@opentelemetry/instrumentation-memcached": "~0.23.0",
-    "@opentelemetry/instrumentation-mongodb": "~0.23.0",
-    "@opentelemetry/instrumentation-mysql": "~0.23.0",
-    "@opentelemetry/instrumentation-mysql2": "~0.23.0",
-    "@opentelemetry/instrumentation-net": "~0.23.0",
-    "@opentelemetry/instrumentation-pg": "~0.23.0",
-    "@opentelemetry/instrumentation-pino": "~0.23.0",
-    "@opentelemetry/instrumentation-redis": "~0.23.0",
-    "@opentelemetry/instrumentation-winston": "~0.24.0",
-    "opentelemetry-instrumentation-amqplib": "~0.23.0",
-    "opentelemetry-instrumentation-aws-sdk": "~0.23.0",
-    "opentelemetry-instrumentation-elasticsearch": "~0.23.0",
-    "opentelemetry-instrumentation-kafkajs": "~0.23.0",
-    "opentelemetry-instrumentation-mongoose": "~0.23.0",
-    "opentelemetry-instrumentation-sequelize": "~0.23.0",
-    "opentelemetry-instrumentation-typeorm": "~0.23.0"
+    "@opentelemetry/instrumentation-bunyan": "~0.25.0",
+    "@opentelemetry/instrumentation-cassandra-driver": "~0.25.0",
+    "@opentelemetry/instrumentation-dns": "~0.25.0",
+    "@opentelemetry/instrumentation-express": "~0.25.0",
+    "@opentelemetry/instrumentation-graphql": "~0.25.0",
+    "@opentelemetry/instrumentation-grpc": "~0.24.0",
+    "@opentelemetry/instrumentation-hapi": "~0.25.0",
+    "@opentelemetry/instrumentation-http": "~0.24.0",
+    "@opentelemetry/instrumentation-ioredis": "~0.25.0",
+    "@opentelemetry/instrumentation-knex": "~0.25.0",
+    "@opentelemetry/instrumentation-koa": "~0.25.0",
+    "@opentelemetry/instrumentation-memcached": "~0.25.0",
+    "@opentelemetry/instrumentation-mongodb": "~0.25.0",
+    "@opentelemetry/instrumentation-mysql": "~0.25.0",
+    "@opentelemetry/instrumentation-mysql2": "~0.25.0",
+    "@opentelemetry/instrumentation-net": "~0.25.0",
+    "@opentelemetry/instrumentation-pg": "~0.25.0",
+    "@opentelemetry/instrumentation-pino": "~0.25.0",
+    "@opentelemetry/instrumentation-redis": "~0.25.0",
+    "@opentelemetry/instrumentation-winston": "~0.25.0",
+    "opentelemetry-instrumentation-amqplib": "~0.25.0",
+    "opentelemetry-instrumentation-aws-sdk": "~0.25.0",
+    "opentelemetry-instrumentation-elasticsearch": "~0.25.0",
+    "opentelemetry-instrumentation-kafkajs": "~0.25.0",
+    "opentelemetry-instrumentation-mongoose": "~0.25.0",
+    "opentelemetry-instrumentation-sequelize": "~0.25.0",
+    "opentelemetry-instrumentation-typeorm": "~0.25.0"
```

Comment:
In npm the 0.25.0 versions are tagged as next as opposed to latest (e.g. instrumentation-http), is this fine?

Reply:
```shell
(test/e2e-tests)$ npm info @opentelemetry/instrumentation-knex dist-tags # same for most of the packages
{ latest: '0.25.0' }
(test/e2e-tests)$ npm info @opentelemetry/instrumentation-http dist-tags # same for http and grpc
{ latest: '0.24.0', canary: '0.25.1-alpha.24', next: '0.25.0' }
```

You are right in the context of `instrumentation-http` and `instrumentation-grpc`, and that's why I've left those to `~0.24.0` and only upgraded the others to `~0.25.0`.

Reply:
Ah I missed the 0.24.0